settings.sh

```diff
@@ -1,18 +1,20 @@
 #!/bin/bash
```

Comment:
settings.sh should be in .gitignore?

Reply:
Eventually, yes.  It's convenient to have it for now, though, while we get this initial version to work.